uint32          id
float32         center_x
float32         center_y
sensor_msgs/Image           image
Vision_Features features
